src/lib/schema.js

```diff
@@ -83,6 +83,17 @@ export function getMetricFromKey(key, bucket) {
 }
 
 /**
+* Returns metric from a state key
+* @param {string} key - schema key
+* @param {string} bucket - bucket name
+* @return {string} metric - S3 metric
+*/
+export function getMetricFromStateKey(key, bucket) {
+    // s3:buckets:foo:storageUtilized
+    return key.slice(11).replace(`${bucket}:`, '');
```

Comment:
Updated. That's much more intuitive—I will eventually update those other methods using the same pattern. Thanks!